oggsound/src/OgreOggSoundManager.cpp

```diff
@@ -1572,22 +1571,45 @@ namespace OgreOggSound
 			return false;
 		}
 
-		if (alIsFilter(filter) && ((filterType==AL_FILTER_LOWPASS) || (filterType==AL_FILTER_HIGHPASS) || (filterType==AL_FILTER_BANDPASS) ))
+		if (alIsFilter(filter))
 		{
 			alFilteri(filter, AL_FILTER_TYPE, filterType);
 			if (alGetError() != AL_NO_ERROR)
 			{
-				Ogre::LogManager::getSingleton().logMessage("*** Filter not supported!");
+				Ogre::LogManager::getSingleton().logMessage("*** Unable to set filter type!");
 				return false;
 			}
 			else
 			{
-				// Set properties
-				alFilterf(filter, AL_LOWPASS_GAIN, gain);
-				alFilterf(filter, AL_LOWPASS_GAINHF, hfGain);
-				mFilterList[fName]=filter;
+				switch(filterType)
+				{
+					case AL_FILTER_LOWPASS:
+						// Set lowpass filter properties
+						alFilterf(filter, AL_LOWPASS_GAIN, gain);
+						alFilterf(filter, AL_LOWPASS_GAINHF, hfGain);
+						mFilterList[fName]=filter;
+					break;
+					case AL_FILTER_HIGHPASS:
+						// Set highpass filter properties
+						alFilterf(filter, AL_LOWPASS_GAIN, gain);
+						alFilterf(filter, AL_HIGHPASS_GAINLF, lfGain);
+						mFilterList[fName]=filter;
+					break;
+					case AL_FILTER_BANDPASS:
+						// Set bandpass filter properties
+						alFilterf(filter, AL_LOWPASS_GAIN, gain);
+						alFilterf(filter, AL_BANDPASS_GAINLF, lfGain);
+						alFilterf(filter, AL_LOWPASS_GAINHF, hfGain);
+						mFilterList[fName]=filter;
+					break;
```

Comment:
```suggestion
					break;
					default:
					    Ogre::LogManager::getSingleton().logError("OggSound - Unknown Filter Type");
					break;
```

Reply:
You left me thinking how the logging should be, what do you think about something like this:
_"OgreOggSoundManager::createEFXFilter() : Unknown Filter Type"_

Reply:
Nevermind, I see that those messages are the norm.

So there is another inconsistency to solve here... patch incoming haha